src/containers/TextInputContainer.js

```diff
@@ -1,27 +1,59 @@
-import React from 'react';
+import React,  { Component, } from 'react';
 import { TextInput } from 'react-native';
 import styled from 'styled-components';
 import theme from '../styles/theme.style.js';
 import PropTypes from 'prop-types';
 
-export function TextInputContainer(props) {
-  return (
-    <StyledTextInput textColor={props.captionColor}>
-      {props.children}
-    </StyledTextInput>
-  );
+class TextInputContainer extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <Container>
+        <Label>{this.props.labelName}</Label>
+        <StyledInputContainer>
+          <StyledTextInput editable={this.props.isConfirmed} placeholder={this.props.placeholder}></StyledTextInput>
+        </StyledInputContainer>
+      </Container>
+    );
+  }
 }
 
 // PROP-TYPES
 TextInputContainer.propTypes = {
-  children: PropTypes.element.isRequired,
 };
 
+const Container = styled.View`
+    flexDirection: row;
```

Comment:
(again not sure if it's github) but indent might seem off compared to the ones below?